src/migrations/versions/f654d7e4fcc7_on_delete_cascade_for_jobagent_and_match.py

```diff
@@ -0,0 +1,55 @@
+"""on delete cascade for jobagent and match
+Revision ID: f654d7e4fcc7
+Revises: 6b495d5a4855
+Create Date: 2024-10-17 07:16:34.262079
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "f654d7e4fcc7"
+down_revision = "6b495d5a4855"
```

Comment:
The order of migrations is wrong. This will run after the attempted jobs removal.

I've tested it:

```
mquery-dev-web-1       | WARNING:  StatReload detected changes in 'tasks.py'. Reloading...
mquery-dev-web-1       | INFO:     Started server process [24]
mquery-dev-web-1       | INFO:     Waiting for application startup.
mquery-dev-web-1       | INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
mquery-dev-web-1       | INFO  [alembic.runtime.migration] Will assume transactional DDL.
mquery-dev-web-1       | INFO  [alembic.runtime.migration] Running upgrade dbb81bd4d47f -> 6b495d5a4855, add jobstatus
mquery-dev-web-1       | Revision ID: 6b495d5a4855
mquery-dev-web-1       | Revises: dbb81bd4d47f
mquery-dev-web-1       | Create Date: 2024-10-15 08:17:30.036531
mquery-postgres-1      | 2024-10-18 14:31:24.310 UTC [51] ERROR:  update or delete on table "job" violates foreign key constraint "match_job_id_fkey" on table "match"
mquery-postgres-1      | 2024-10-18 14:31:24.310 UTC [51] DETAIL:  Key (internal_id)=(34) is still referenced from table "match".
mquery-postgres-1      | 2024-10-18 14:31:24.310 UTC [51] STATEMENT:  DELETE FROM job WHERE status = 'removed';
```

Reply:
Both migrations can be probably merged into one to fix this.

Reply:
Yeah, sounds good. A benefit of one big migration is that if something goes wrong then a rollback is performed (so the db is not stuck in an invalid state).